src/cache/cache.h

```diff
@@ -0,0 +1,79 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#ifndef CACHE_H
+#define CACHE_H
+
+#include "./cache_includes.h"
+#include "./lru_cache_manager/lru_cache_manager.h"
+#include "./rax_cache_storage/rax_cache_storage.h"
+#include "xxhash_query_hash/xxhash_query_hash.h"
+
+/**
+ * A struct to for read queries cache storage. 
```

Comment:
Key-value cache with a hard limit on the number of items stored,
Keep the description simple, without going into technical details.

Reply:
done